### PR TITLE
Consumers count tkratky

### PR DESCRIPTION
**Pull request by tkratky**

### CODE REVIEW

File: systemtests/src/main/java/io/enmasse/systemtest/BrokerManagement.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package io.enmasse.systemtest;
```
Comment (lulf) [inline systemtests/src/main/java/io/enmasse/systemtest/BrokerManagement.java]: I think you can just merge this with ArtemisManagement as it doesn't really add anything by being in a separate  class.

File: systemtests/src/test/java/io/enmasse/systemtest/TestBase.java [hunk 2]
```diff
@@ -71,12 +77,18 @@ public void setup() throws Exception {
         addressSpaceList = new ArrayList<>();
         amqpClientFactory = new AmqpClientFactory(openShift, environment, null, username, password);
         mqttClientFactory = new MqttClientFactory(openShift, environment, null, username, password);
+        managementAmqpClientFactory = new AmqpClientFactory(openShift,
+                environment,
+                null,
+                managementCredentials.getUsername(),
+                managementCredentials.getPassword());
```
Comment (lulf) [inline systemtests/src/test/java/io/enmasse/systemtest/TestBase.java]: Not sure why you'd create another factory here. The username/password is just about defaults. You can create clients where you set the username/password explicitly to something different instead.

File: systemtests/src/test/java/io/enmasse/systemtest/TestBaseWithDefault.java [hunk 3]
```diff
@@ -74,8 +76,21 @@ public void setupDefault() throws Exception {
...
+        getKeycloakClient().createUser(defaultAddressSpace.getName(),
+                managementCredentials.getUsername(),
+                managementCredentials.getPassword());
+
+        createGroup(defaultAddressSpace, "admin");
+        joinGroup(defaultAddressSpace, "admin", managementCredentials.getUsername());
+
         amqpClientFactory = new AmqpClientFactory(openShift, environment, defaultAddressSpace, username, password);
         mqttClientFactory = new MqttClientFactory(openShift, environment, defaultAddressSpace, username, password);
+        managementAmqpClientFactory = new AmqpClientFactory(openShift,
```
Comment (lulf) [inline systemtests/src/test/java/io/enmasse/systemtest/TestBaseWithDefault.java]: Same as above, shouldn't be needed.

File: systemtests/src/main/java/io/enmasse/systemtest/BrokerManagement.java (hunk 1 above)
Comment (tkratky) [reply systemtests/src/main/java/io/enmasse/systemtest/BrokerManagement.java]: I would like to implement logic for apache activemq broker in the future (not sure if this broker will be used in enmasse)

File: systemtests/src/test/java/io/enmasse/systemtest/TestBase.java (hunk 2 above)
Comment (tkratky) [reply systemtests/src/test/java/io/enmasse/systemtest/TestBase.java]: that's true, thank you

File: systemtests/src/main/java/io/enmasse/systemtest/BrokerManagement.java (hunk 1 above)
Comment (lulf) [reply systemtests/src/main/java/io/enmasse/systemtest/BrokerManagement.java]: It's not a big deal, but me personal preference is that such a refactoring should be done when its needed. I'm fine with the current approach.

**Pull request merged**